ikasaneip/visualisation-dashboard/src/main/resources/ikasanapp_ja.properties

```diff
@@ -213,3 +213,4 @@ label.status-paused=イカさんは最近とても忙しい
 label.status-startPause=イカさんは最近とても忙しい
 label.status-stopped=イカさんは最近とても忙しい
 status.click-for-module-status=イカさんは最近とても忙しい
+message.resubmission-event-action=イカさんは最近とても忙しい
```

Comment:
Arent there few keys missing in JP version

Reply:
Someone else’s job. I have no tags in the German file. And there is no Chinese file in existence yet.